build.properties

```diff
@@ -12,5 +12,5 @@ wpilib.pluginURL=http://first.wpi.edu/FRC/roborio/release/eclipse/plugins/edu.wp
 
 # If set, and set to true or yes, will download and install the CTRE User Library
 userLibInstallCtre=true
-userLibInstallCtre.URL=http://www.ctr-electronics.com/downloads/lib/CTRE_FRCLibs_NON-WINDOWS.zip
+userLibInstallCtre.URL=http://www.ctr-electronics.com/downloads/lib/CTRE_FRCLibs_NON-WINDOWS_v4.4.1.9.zip
```

Comment:
This causes some issues with the CTRE dependencies.  I'm going to work it out tomorrow night and we'll push a fix in a separate branch.  Leave this for now, but you should probably expect merge conflicts in this file in the future.